### PR TITLE
CNV-46339: Updated RHEL OS version number

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.16+

Issue: [CNV-46339](https://issues.redhat.com//browse/CNV-46339)

Link to docs preview: Step 4 in module https://84226--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-using-dpdk-with-sriov.html#virt-configuring-vm-dpdk_virt-using-dpdk-with-sriov

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by orelmisan — APPROVED** [review]
Thank you @sjhala-ccs

File: modules/virt-configuring-vm-dpdk.adoc [hunk 1]
```diff
@@ -75,7 +75,7 @@ spec:
 $ oc apply -f <file_name>.yaml
 ----
 
-. Configure the guest operating system. The following example shows the configuration steps for {op-system-base} 8 OS:
+. Configure the guest operating system. The following example shows the configuration steps for {op-system-base} 9 OS:
```
Comment (dfitzmau) [inline modules/virt-configuring-vm-dpdk.adoc:78]: Although it's not part of your direct changes, "OS" should not be used: 

Screenshot from 2024-10-30 14-18-43

Comment (dfitzmau) [reply modules/virt-configuring-vm-dpdk.adoc:78]: https://redhat-documentation.github.io/supplementary-style-guide/#operating-system

**Review by dfitzmau — APPROVED** [review]
/remove-label peer-review-in-progress

**Review by yossisegev — APPROVED** [review]

File: modules/virt-configuring-vm-dpdk.adoc (hunk 1 above)
Comment (sjhala-ccs) [reply modules/virt-configuring-vm-dpdk.adoc:78]: Done

**Pull request merged**